pennylane/templates/subroutines/approx_time_evolution.py

```diff
@@ -24,12 +24,6 @@ class ApproxTimeEvolution(Operation):
     r"""Applies the Trotterized time-evolution operator for an arbitrary Hamiltonian, expressed in terms
     of Pauli gates.
 
-    .. note::
-
-        This operation is a special case of :class:`~pennylane.TrotterProduct` and will be deprecated in
-        future releases, please use :class:`~pennylane.TrotterProduct` as the preferred operation for this
-        functionality.
-
```

Comment:
Why this is removed?

Reply:
I had a quick chat with @trbromley, and we discussed that we might not want to point users away from ApproxTimeEvolution just yet, especially since the conversion is a bit non-trivial. Instead if someone is using this, we let them continue using it. 

When we deprecate this, we will add a more detailed message about how to do the conversion (like in the TrotterProduct docs).